options/options.c

```diff
@@ -409,6 +411,10 @@ const struct m_sub_options mp_osd_render_sub_opts = {
         {"osd-selected-color", OPT_COLOR(osd_selected_color)},
         {"osd-selected-outline-color", OPT_COLOR(osd_selected_outline_color)},
         {"force-rgba-osd-rendering", OPT_BOOL(force_rgba_osd)},
+        {"osd-prune-delay", OPT_DOUBLE(osd_ass_prune_delay), M_RANGE(-1.0, LLONG_MAX / 1000.0)},
```

Comment:
`LONG_MAX` will work too

Reply:
the libass function accepts `long long` so this is probably more correct

Reply:
it builds locally for me, but M_RANGE() is cast to double, so i dont think we can use long here anyway. it complains about `../options/options.c:330:76: warning: implicit conversion from 'long' to 'double' changes value from 9223372036854775807 to 9223372036854775808 [-Wimplicit-const-int-float-conversion]`

Reply:
int in this case. Anyway this can be arbitrary limit, it doesn't really matter.

Reply:
hardcoded to 10000.0 because nobody should ever want to set a higher value